docs/supporting/faq.md

```diff
@@ -274,6 +274,12 @@ Microsoft [ended support for Windows 7](https://learn.microsoft.com/lifecycle/pr
 
 VS Code will no longer provide product updates or security fixes on Windows 7 and VS Code [version 1.70.3](https://code.visualstudio.com/updates/v1_70) is the last available release for Windows 7 users. You can learn more about upgrading your Windows version at [support.microsoft.com](https://support.microsoft.com/windows/windows-7-support-ended-on-january-14-2020-b75d4580-2cc7-895a-2c9c-1466d9a53962).
 
+## Can I run VS Code on old macOS versions?
+
+VS Code desktop versions starting with 1.76 (February 2023) will no longer run on macOS Sierra (version 10.12 and older) and you will need to upgrade to a newer macOS version to use later versions of VS Code.
```

Comment:
@isidorn our plan is to push preliminary notifications for January milestone via https://github.com/microsoft/vscode/pull/171598 and we only disable updates from February which means `1.76` will be the last running versions. So maybe the above should be `1.77` instead ?

Reply:
Correct. I made an off by one math error.
Let me fix.